ssh/README.mdx

```diff
@@ -28,17 +28,13 @@ you haven't already.
 
 Start by configuring SSH access on your laptop:
 
-<ContentLink icon={<FileWithFlippedCornerIcon />} href="client">
-  Client Quickstart Guide &gt;
-</ContentLink>
+**[Client Quickstart Guide](./client.mdx)**
 
 ## Step 2: Register Hosts
 
 Register your hosts with Smallstep SSH:
 
-<ContentLink icon={<FileWithFlippedCornerIcon />} href="hosts">
-  Host Quickstart Guide &gt;
-</ContentLink>
+**[Host Quickstart Guide](./client.mdx)**
```

Comment:
@tashian - Should this be something like (./host.mdx)** versus client? It appears to be the same as the line 31 change to (./client.mdx)**